civis/tests/test_parallel.py

```diff
@@ -327,6 +343,22 @@ def _get_container(job_id):
     mock_make_factory.assert_called_once_with(**expected_kwargs)
 
 
+@mock.patch.object(civis.parallel, 'make_backend_factory')
+def test_infer_in_child_job(mock_make_factory, mock_child_job):
+    # Verify that infer_backend_factory doesn't include CIVIS_PARENT_JOB_ID and
+    # CIVIS_PARENT_RUN_ID since those will be automatically added later.
+    mock_client = mock.MagicMock()
```

Comment:
Suggestion: Using `create_client_mock` will give you an autospecced APIClient mock, which should error on things like invalid / misspelled endpoint names.
```suggestion
    mock_client = create_client_mock()
```